modules/disabling-catalogsource-objects.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * admin/olm-cs-podsched.adoc
+
+:_content-type: PROCEDURE
+[id="disabling-catalogsource-objects_{context}"]
+= Disabling default CatalogSource objects at a local level
+
+You can apply persistent changes to a `CatalogSource` object, such as catalog source pods, at a local level, by disabling a default `CatalogSource` object. Consider the default configuration in situations where the default `CatalogSource` object's configuration does not meet your organization's needs. By default, if you modify fields in the `spec.grpcPodConfig` section of the   `CatalogSource` object, the Marketplace Operator automatically reverts these changes.
+
+The Marketplace Operator, `openshift-marketplace`, manages the default custom resources (CRs) of the `OperatorHub`. The `OperatorHub` manages `CatalogSource` objects. 
+
+To apply persistent changes to `CatalogSource` object, you must first disable a default `CatalogSource` object.
+
+.Procedure
+
+* To disable all the default `CatalogSource` objects at a local level, issue the following command:
+
+[source,terminal]
```

Comment:
Needs a `+` to attach the block to the step. 

Re: `run` vs `issue`, [OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) specify `run` or `enter`. If you want to avoid `run`, I would suggest using `enter` so we can (try to :wink:) be consistent

```suggestion
* To disable all the default `CatalogSource` objects at a local level, issue the following command:
+
[source,terminal]
```

Reply:
Nicely spotted. I missed that guidance; cannot argue with that! I swapped "issue" for "enter".

I also added the plus (+) symbols.

Reply:
I cannot reply to your other comment as it's outdated. But the text for `olm-cs-podsched.adoc` now matches `disabling-catalogsource-objects.adoc`. I missed this. Well spotted.